primitive-types/impls/num-traits/src/lib.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020 Parity Technologies
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+//! num-traits support for uint.
+
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
I don't understand why is this needed?

Reply:
Bit of a leftover from moving `from_str_radix` into the `uint` crate. I don't believe this crate needs it ever, but there is a feature that enables `std` in `uint` and `num-traits`.